### PR TITLE
Applying filter on all db changes since 0 takes too long

### PR DESCRIPTION
**Pull request by jasonpet**
Updating provider to use a view for loading all existing triggers into memory on startup.  Will continue to use the filter for follow.

### CODE REVIEW

**Review by jberstler — APPROVED** [review]
LGTM. @csantanapr can you also have a look?

File: provider/lib/utils.js [hunk 1]
```diff
@@ -216,25 +210,26 @@ module.exports = function(
     this.initAllTriggers = function() {
         var method = 'initAllTriggers';
 
-        logger.info(method, 'resetting system from last state');
+        //follow the trigger DB
+        utils.setupFollow('now');
 
-        triggerDB.changes({ since: 0, include_docs: true, filter: ddname + '/' + filter, worker: utils.worker }, (err, changes) => {
+        logger.info(method, 'resetting system from last state');
+        triggerDB.view(viewDDName, triggersByWorker, {reduce: false, include_docs: true}, function(err, body) {
```
Comment (csantanapr) [inline provider/lib/utils.js]: use `key:"worker0"` that will reduce the docs you get back only for the current worker

File: provider/lib/utils.js [hunk 2]
```diff
@@ -216,25 +210,26 @@ module.exports = function(
...
             if (!err) {
-                changes.results.forEach(function (change) {
-                    var triggerIdentifier = change.id;
-                    var doc = change.doc;
+                body.rows.forEach(function (trigger) {
+                    var triggerIdentifier = trigger.id;
+                    var doc = trigger.doc;
 
-                    if (!doc.status || doc.status.active === true) {
+                    if (trigger.key === utils.worker && (!doc.status || doc.status.active === true) && !(triggerIdentifier in utils.triggers)) {
```
Comment (csantanapr) [inline provider/lib/utils.js]: by using key in the view, no need to check the worker

**Review by csantanapr — APPROVED** [review]
LGTM

**Pull request merged**